Patch added by Enrique Robledo Arnuncio <era@debian.org> in version 0.6.1-1, see #263437.

--- ../orig/freqtweak-0.7.0~cvs20070605.orig/configure.ac	2005-11-10 05:17:04.000000000 +0100
+++ configure.ac	2007-06-05 10:23:41.000000000 +0200
@@ -353,14 +353,17 @@
       [ if test "x$enable_debug" != "xno" ; then
 	  CXXFLAGS="-g -Wall -D_REENTRANT $FFTW_CFLAGS $JACK_CFLAGS $WX_CFLAGS $XML_CFLAGS $SIGCPP_CFLAGS"
 	  CFLAGS="$CXXFLAGS"
+	  CPPFLAGS="$CPPFLAGS $XML_CPPFLAGS"
         else
 	  CXXFLAGS="$FFTW_CFLAGS $JACK_CFLAGS $WX_CFLAGS $XML_CFLAGS $SIGCPP_CFLAGS -Wall -D_REENTRANT $ARCH_CFLAGS"
 	  CFLAGS="$CXXFLAGS"
+	  CPPFLAGS="$CPPFLAGS $XML_CPPFLAGS"
 	fi
       ],
 	[
 	  CXXFLAGS="$FFTW_CFLAGS $JACK_CFLAGS $WX_CFLAGS $XML_CFLAGS $SIGCPP_CFLAGS -Wall -D_REENTRANT $ARCH_CFLAGS"
 	  CFLAGS="$CXXFLAGS"
+	  CPPFLAGS="$CPPFLAGS $XML_CPPFLAGS"
         ]
 )
 
